src/configuration/configuration.ts

```diff
@@ -1,3 +1,3 @@
 export const configuration = {
-	production: true
+	production: false
```

Comment:
need to change back to true?

Reply:
Development should by default to be false..

Reply:
but it goes into production, isn't it?